packages/frontend/src/router/router.tsx

```diff
@@ -101,27 +89,26 @@ class MobxRouter {
       return params;
     };
     this.routes.unshift({ match, component });
-  };
+  }
 
-  @action
-  addRedirect = (newRedirect: Redirect) => {
+  @action.bound
```

Comment:
Why is this changed?

Reply:
A very silly reason: since its a JSX file, TypeScript was trying to interpret my `<T>` argument as the beginning of a JSX block when I wrote `addRedirect = <T>(newRedirect:Redirect)`